pep_sphinx_extensions/pep_zero_generator/constants.py

```diff
@@ -45,4 +45,10 @@
 .. _specifications: https://packaging.python.org/en/latest/specifications/
 .. _PyPA specification update process: https://www.pypa.io/en/latest/specifications/#specification-update-process
 """,
+    "release": """\
+A PEP is written to specify the release cycle for each feature release of Python.
+See the `developer's guide`_ for more information.
+
+.. _developer's guide: https://devguide.python.org/
```

Comment:
```suggestion
.. _developer's guide: https://devguide.python.org/devcycle/
```

Would be good to link the relevant page directly.